torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5575,22 +5575,20 @@ def skips_mvlgamma(skip_redundant=False):
 # Class `MvlGammaInfo` already contains the basic information related to the operator,
```

Comment:
I think the pR needs to update this comment if it's switching the class for a factory function